plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/related-by-control.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	CheckboxControl,
+	PanelBody,
+	// @ts-expect-error missing types.
+	__experimentalVStack as VStack,
```

Comment:
Are we using VStack in this file?

Reply:
Thanks for catching that! I initially used VStack but later simplified it to a regular `div` element since I didn't need any of VStack's specific layout features. I will remove it.